pkg/tagger/collectors/static_extract.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+// +build docker kubelet
```

Comment:
Do we need to guard with these build flags? Does look pretty generic here? Maybe some other dependency requirement to have these build flags on?

Reply:
i had in the back of my mind any potential downsides to having `DD_TAGS` picked up as host tags as well as the static collector, but even then this isn't the right way to guard against that. going to take ahmed's suggestion and only use this check on fargate, as those are the situations where the hostname is empty and thus these tags aren't added to the metadata